e2e/tests/apiproduct-apikeys-tab.spec.ts

```
@@ -0,0 +1,376 @@
+    // Cleanup temp file
+    try {
+      fs.unlinkSync(tmpFile);
+    } catch (e) {
+      // Ignore cleanup errors
+    }
+  }
+}
+
+async function deleteNamespace(name: string) {
+  try {
+    await execAsync(`kubectl delete namespace ${name} --ignore-not-found=true --wait --timeout=10s`);
```

ESLint

[error] Replace ``kubectl·delete·namespace·${name}·--ignore-not-found=true·--wait·--timeout=10s`` with `⏎······`kubectl·delete·namespace·${name}·--ignore-not-found=true·--wait·--timeout=10s`,⏎····`

(prettier/prettier)